java-client/src/test/resources/log4j.properties

```diff
@@ -17,15 +17,9 @@
 # under the License.
 #
 
-meta_servers = 127.0.0.1:34601,127.0.0.1:34602,127.0.0.1:34603
-operation_timeout = 5000
-async_workers = 4
-enable_perf_counter = false
-perf_counter_tags = cluster=onebox,app=unit_test
-push_counter_interval_secs = 10
-meta_query_timeout = 5000
-auth_protocol =
-kerberos_service_name =
-kerberos_service_fqdn =
-kerberos_keytab =
-kerberos_principal =
```

Comment:
Is it necessary to also move these properties to `pegasus.properties` under `src/test/resources` as a configuration example for developers ?
```properties
meta_query_timeout = 5000
auth_protocol =
kerberos_service_name =
kerberos_service_fqdn =
kerberos_keytab =
kerberos_principal =
```

Reply:
Done.